package.json

```diff
@@ -45,7 +45,7 @@
     "aigle": "git+https://github.com/suguru03/aigle#8739846ba9d4cfc116e1546da1181c73564cae0b",
     "aware": "^0.3.1",
     "bignumber.js": "^9.0.0",
-    "dvf-utils": "git+https://github.com/DeversiFi/dvf-utils#77692b6e5d45f6051e9263fab209589db145012c",
+    "dvf-utils": "git+https://github.com/DeversiFi/dvf-utils#d047a36547c17991e012d783850360a77c070c12",
```

Comment:
probably best to update to the latest commit from master (d047a36547c17991e012d783850360a77c070c12) as it is also being updated on another #181

Reply:
I'll merge this first since the other PR still needs to be updated